docs/usage/SYNC_OPTIONS.md

```diff
@@ -1,10 +1,10 @@
 # Sync Options
 
-#### `errorCallBack`
+#### `errorCallback`
 a callback that is called whenever an error event occurs during the sync process. It contains information about the 
```

Comment:
Maybe also update this here as we provide now more info, and also for warningCallback.

Reply:
928164a4